documentation/whatsnew/batch/batch-news-4.3.0.Beta2.adoc

```diff
@@ -0,0 +1,26 @@
+= Batch Tools What's New in 4.3.0.Beta2
+:page-layout: whatsnew
+:page-component_id: batch
+:page-component_version: 4.3.0.Beta2
+:page-product_id: jbt_core 
+:page-product_version: 4.3.0.Beta2
+
+== Hyprelinks for @BatchProperty
+
+It's now possible to navigate (Ctrl+Click) from injection points annotated @BatchProperty to the corresponding property references:
+
+image::images/4.3.0.Beta2/openon.gif[]
+
+related_jira::JBIDE-19700[]
+
+== Navigation to Batch artifacts from properties editor 
+
+To navigate from Properties editor to the corresponding batch artifact click on the icon at the property name and select Open Artifact.
+
+image::images/4.3.0.Beta2/open.png[]
+
+If there is no such artifact yet then instead of Open Artifact there will be Create Artifact. Click on this menu to open New Batch Artifact wizard. 
```

Comment:
This sentence is a bit hard to parse.

suggestion: "If there are no artifact yet then there will be a 'Create artifact' link which when clicked opens 'New Batch Artifact' wizard" ? 

btw. can the link be a bit more specific than "create artifact" ? that is like writing "Create thing" .....is it not prety specific thing there needs to be created ? ie. a job definition ?